services/iplant-email/test/iplant_email/core_test.clj

```diff
@@ -0,0 +1,7 @@
+(ns iplant-email.core-test
```

Comment:
this file move is because this seems to be the more common version now; the old test didn't have any functional components anyway